pages/common/git-pull.md

```diff
@@ -0,0 +1,11 @@
+# git pull
+
+> Fetch branch from a remote repo and merge it to current repo
+
+- Download the remote-tracking branch from default remote and merge it
```

Comment:
Also, "default remote" --> "default remote repo(sitory)", just to be explicit and leave no room for doubts.